hosting/learnings-from-2025.md

```diff
@@ -0,0 +1,80 @@
+# Learnings from the 2025 Hackathon
+
+All in all, this was surprisingly successful. There were some technical issues though.
+
+## Overall organization
+
+* Andrew, Pier-Luigi, Sara and Yuting did a great job at holding things together
+* Bjorn did a great job at shaking up structures and pushing things forward
+* Tobi, Lukas, Mark M, and others made technical magic possibble.
+
+## Schedule
+
+## Inter-node communication
+
+### Mattermost
+
+* Works
+* Invite more people earlier in the process
+* China uses WeChat for most communication (in Chinese, but with auto-translation)
+
+### Zoom calls during the preparation phase
+
+* Double Telcos are tiring but necessary.
+* Did anybody read the nodes / watch the recordings?
+
+### Zoom calls during the hackathon
+
+* Did not really line up with the schedules (at least at some nodes). Would need earlier planning to be incorporated into the local schedules.
+
+### GitHub and Code sharing
+
+* GitHub seems to provide a bit of a barrier for sharing code.
+* Need a concept that scales to 500 people contributing at the same time without established teams or time for learning.
+
+## Technical aspects
+
+### Technical Team
+
+* Actually have a real team with members from all nodes
+* Meet with this team at EGU/AGU if possible at all
+* Distribute tech team members more widely to foster connections?
+* Might be worth hiring one person to oversee things / fully commit to running this
+* Have a clear contact for each node who leads the preparations and actually is there during the hackathon
+
+### Dataset preparation
+
+* Provide default chunk settings for 2D/3D datasets
+* Zarr with small (regional) Chunks works well.
+    * The IFS GRIB datasets were very popular in China despite access via EERIE cloud being difficult.
```

Comment:
Do you know if this was because or despite IFS being GRIB?

From what I have seen most participants didn't care (or understand) the different data formats behind the catalog–which is a success on it's own!

This means that we should use "data backends" that scale well for the data providers. Observing the issue reports on the eerie.cloud, the GRIB datasets were not so popular